app/questionnaire/placeholder_transforms.py

```diff
@@ -31,8 +31,27 @@ def format_list(list_to_format):
         return formatted_list
 
     @staticmethod
-    def concatenate_list(list_to_concatenate, delimiter):
-        return delimiter.join(list_to_concatenate)
+    def remove_empty(list_to_filter):
```

Comment:
I think we should make this method's name a bit more descriptive, maybe `remove_all_empty` or `remove_all_empty_items`.